taxonomy-editor-frontend/src/pages/root-nodes/index.tsx

```diff
@@ -0,0 +1,207 @@
+import { useState, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+
+import {
+  Typography,
+  Snackbar,
+  Alert,
+  Box,
+  Stack,
+  IconButton,
+} from "@mui/material";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import EditIcon from "@mui/icons-material/Edit";
+import AddBoxIcon from "@mui/icons-material/AddBox";
+import Dialog from "@mui/material/Dialog";
+import DialogTitle from "@mui/material/DialogTitle";
+
+import CreateNodeDialogContent from "./CreateNodeDialogContent";
+import useFetch from "../../components/useFetch";
+import { createBaseURL } from "../editentry/createURL";
+import { toTitleCase } from "../../utils";
+import { greyHexCode } from "../../constants";
+
+type DataType = {
```

Comment:
Can we change the name of this type? Seems too abstract.

Reply:
done, added backend type